.gitignore

```
@@ -6,10 +6,13 @@
 /pkg/
 /spec/reports/
 /tmp/
+/vendor/
 *.bundle
 *.so
 *.o
 *.a
 mkmf.log
 npm-debug.log
 *.gem
+node_modules
+package-lock.json
```

I think the `Gemfile.lock` could be ignored in the same way the `package-lock.json` is ignored.

_I used to include the lock files in my packages for testing, but to be honest I haven't ever run into a situation where that proved useful to troubleshoot differences between local testing and continuous testing, so maybe it's not that useful after all. YMMV_

```suggestion
package-lock.json
Gemfile.lock
```

---

In the realm of Python they suggest to commit Pipfile.lock as well, but as you wrote - I usually don't see it that useful for libraries. I might do what you suggest as a next step in a separate PR.